cost/google/cud_report/Google_Committed_UseDiscount_Report.pt

```diff
@@ -0,0 +1,124 @@
+name "Google Committed Use Discount (CUD) Report"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report on all Google CUDs that have been purchased. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/google/cud_report) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Cost"
+severity "low"
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_google_project" do
+  type "string"
+  label "Google Cloud Project"
+  allowed_pattern /^[0-9a-z:\.-]+$/
+end
+
+parameter "param_google_status" do
```

Comment:
maybe name param_cud_status - it's the status of the CUD, not the google project

Reply:
Changed "param_google_status" to "param_cud_status"